### PR TITLE
Fix: bootstrap: -N option setup the current node and peers all together 

### PR DESCRIPTION
**Pull request by liangxin1300**
#### Enable -N option on init node
```
tbw-1:~ # crm cluster init -N tbw-1 -N tbw-2 -N tbw-3 -y
INFO: Loading "default" profile from /etc/crm/profiles.yml
INFO: SSH key for root does not exist, hence generate it now
INFO: SSH key for hacluster does not exist, hence generate it now

INFO: Configuring SSH passwordless with root@tbw-2
root@tbw-2's password: 
INFO: SSH key for root does not exist, hence generate it now
INFO: Configuring SSH passwordless with root@tbw-3
root@tbw-3's password: 
INFO: SSH key for root does not exist, hence generate it now

INFO: Configuring csync2
INFO: Generating csync2 shared key (this may take a while)...done
INFO: csync2 checking files...done
INFO: Configuring corosync (unicast)
WARNING: Not configuring SBD - STONITH will be disabled.
WARNING: Hawk not installed - not configuring web management interface.
WARNING: You should change the hacluster password to something more secure!
INFO: Starting pacemaker...done
INFO: Waiting for cluster..............done
INFO: Loading initial cluster configuration
INFO: Done (log saved to /var/log/crmsh/crmsh.log)

INFO: Adding node tbw-2 to cluster
INFO: Running command on tbw-2: crm cluster join -y -c tbw-1
INFO: SSH key for hacluster does not exist, hence generate it now
INFO: Configuring SSH passwordless with hacluster@tbw-1
INFO: Configuring csync2...done
INFO: Merging known_hosts
INFO: Probing for new partitions...done
WARNING: Hawk not installed - not configuring web management interface.
WARNING: You should change the hacluster password to something more secure!
INFO: Starting pacemaker...done
INFO: Waiting for cluster.....done
INFO: Reloading cluster configuration...done
INFO: Done (log saved to /var/log/crmsh/crmsh.log)

INFO: Adding node tbw-3 to cluster
INFO: Running command on tbw-3: crm cluster join -y -c tbw-1
INFO: SSH key for hacluster does not exist, hence generate it now
INFO: Configuring SSH passwordless with hacluster@tbw-1
INFO: Configuring SSH passwordless with hacluster@tbw-2
INFO: Configuring csync2...
...
```
#### Validate -N option
```
# unreachable node
tbw-1:~ # crm cluster init -N tbw-1 -N tbw100 -y
ERROR: cluster.init: host "tbw100" is unreachable: ping: tbw100: Name or service not known
```
```
# duplicated input
tbw-1:~ # crm cluster init -N tbw-1 -N tbw-2 -N tbw-2 -y
ERROR: cluster.init: Duplicated input for -N/--nodes option
```
```
# one of node already started
tbw-1:~ # crm cluster init -N tbw-1 -N tbw-2 -N tbw-3 -y
INFO: Loading "default" profile from /etc/crm/profiles.yml

ERROR: cluster.init: Cluster is currently active on tbw-2 - can't run
```

### CODE REVIEW

**Comment by liangxin1300** [comment]
> -N is a good idea, I agree!
> 
> While revisiting the user experience, I have impression to make "add" as the alias of "join" to simplify the user interface. Otherwise, I (users) will ask what makes the real difference between them? Does is worth to have those difference?

`crm cluster add` is to add other node on init node
`crm cluster join` is to join the cluster on other node

`crm cluster add` using the same codes with `crm cluster init -N` option.
If it makes confuse, I prefer to remove `crm cluster add` completely, since we already have `-N` option on `crm cluster init` side.
And also because these years seems no customer using `crm cluster add`(since no bug from this long broken option)
What do you think @zzhou1 ?

File: crmsh/ui_cluster.py [hunk 1]
```diff
@@ -301,9 +301,8 @@ def looks_like_hostnames(lst):
                             help='Answer "yes" to all prompts (use with caution, this is destructive, especially those storage related configurations and stages. The /root/.ssh/id_rsa key will be overwritten unless the option "--no-overwrite-sshkey" is used)')
         parser.add_argument("-n", "--name", metavar="NAME", dest="cluster_name", default="hacluster",
                             help='Set the name of the configured cluster.')
-        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes",
+        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes", action="append", default=[],
```
Comment (zzhou1) [inline crmsh/ui_cluster.py:304]: Cosmetic improvement for the grammar in UI

```suggestion
        parser.add_argument("-N", "--node", metavar="NODENAME", dest="node", action="append", default=[],
```

File: crmsh/ui_cluster.py [hunk 2]
```diff
@@ -301,9 +301,8 @@ def looks_like_hostnames(lst):
...
                             help='Additional nodes to add to the created cluster. May include the current node, which will always be the initial cluster node.')
```
Comment (zzhou1) [inline crmsh/ui_cluster.py:305]: Cosmetic improvement for the grammar in UI

```suggestion
                            help='The member node of the cluster. Note: the current node is always get initialized during bootstrap in the beginning.')
```

**Review by zzhou1 — CHANGES_REQUESTED** [review]

File: crmsh/bootstrap.py [hunk 3]
```diff
@@ -167,6 +168,21 @@ def _validate_sbd_option(self):
             if self.cluster_is_running:
                 utils.check_all_nodes_reachable()
 
+    def _validate_nodes_option(self):
+        """
+        Validate -N/--nodes option
+        """
+        self.nodes = utils.parse_append_action_argument(self.nodes)
```
[the remark this answers is not in the document]
Comment (liangxin1300) [reply crmsh/bootstrap.py:175]: I don't think so
This happens just at beginning of the process, no harm to exit
What if user's intention is `-N node1 -N node2`, but he input wrong as `-N node1 -N node1`?

Besides that, this behavior is the same with `-i` and `-s` option, for the multi interface and multi sbd device

File: crmsh/ui_cluster.py (hunk 1 above)
Comment (liangxin1300) [reply crmsh/ui_cluster.py]: Changed, except dest changed as node_list

**Review by zzhou1 — APPROVED** [review]

**Pull request merged**